### ISSUE TITLE
French translation changes layout

### ISSUE THREAD

**Issue by fylux** [rust-lang/rust-www#626]
Hi, I've seen that the french translation is the only one that affects the menu layout:
![imagen](https://cloud.githubusercontent.com/assets/1207962/20829838/1c3dd25a-b87e-11e6-86bf-2f205c2a2cb3.png)

I think that it should be changed to have the same style as the rest of the translations. There are two options:

-Change the CSS code of the page (the most scalable option in my opinion).
-Change the translation.
 
Regards.

**Comment by 3442853561** [comment]
I will try to fix it now, thank you for issue.

**Comment by brson** [comment]
It looks ok to me today. The word "Documentation" recently changed to "Install", so maybe that makes it fit.

**Comment by fylux** [comment]
@brson Yes, now it looks okay, but maybe we should still consider the issue to avoid future possible problems.

**Comment by AndrewBrinker** [comment]
There is an effort in progress to redesign the Rust website. This may be something for the redesign to consider, but for now it seems minor enough to not really worry about.

**Comment by Insomgla** [comment]
@fylux Yes we should, because "Install" isn't even a French word. It should be "Installation", but then the problem will occur again.
